README.md

```diff
@@ -86,7 +86,20 @@ When the `NamespaceScope` CR is created/updated, it will:
       resources:
       - '*'
       verbs:
-      - '*'
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - rbac.authorization.k8s.io
+      resources:
+      - roles
+      verbs:
+      - escalate
```

Comment:
let's put on hold this change till we clarify it does not effectively increase the privileged access